MAINTAINERS.md

```diff
@@ -8,31 +8,32 @@
 | ---------------- | ---------------- | ---------------- |
 | Abdel Bakhta     | abdelhamidbakhta | abdelhamidbakhta |
 | Adrian Sutton    | ajsutton         | ajsutton         |
-| Antoine Toulme   | atoulme          | atoulme          |
 | Antony Denyer    | antonydenyer     | antonydenyer     |
+| Antoine Toulme   | atoulme          | atoulme          |
 | Byron Gravenorst | bgravenorst      | bgravenorst      |
 | Daniel Lehrner   | daniellehrner    | daniellehrner    |
 | Diego López León | diega            | diega            |
 | Fabio Di Fabio   | fab-10           | fab-10           |
 | Frank Li         | frankisawesome   | frankliawesome   |
 | Gary Schulte     | garyschulte      | GarySchulte      | 
-| Jason Frame      | jframe           | jframe           |
 | Jiri Peinlich    | gezero           | JiriPeinlich     |
-| Joshua Fernandes | joshuafernandes  | joshuafernandes  |
 | Justin Florentine| jflo             | RoboCopsGoneMad  |
+| Jason Frame      | jframe           | jframe           |
+| Joshua Fernandes | joshuafernandes  | joshuafernandes  |
 | Lucas Saldanha   | lucassaldanha    | lucassaldanha    |
 | Sally MacFarlane | macfarla         | macfarla         |
 | Madeline Murray  | MadelineMurray   | madelinemurray   |
 | Mark Terry       | mark-terry       | m.terry          |
 | Karim Taam       | matkt            | matkt            |
 | Meredith Baxter  | mbaxter          | mbaxter          |
 | Nicolas Massart  | NicolasMassart   | NicolasMassart   |
-| Sajida Zouarhi   | sajz             | SajidaZ          |
-| Simon Dudley     | siladu           | siladu           |
 | Stefan Pingel    | pinges           | pinges           |
-| Taccat Isid      | taccatisid       | taccatisid       |
 | Rai Sur          | RatanRSur        | ratanraisur      |
+| Ryan Jones       | ryanjjones10     | ryanjjones10     |
```

Comment:
This is the only new maintainer added in this PR. Other changes are just shuffling names around.